core/src/test/resources/conf/testApplication.properties

```diff
@@ -0,0 +1,6 @@
+db.driver=org.h2.Driver
+db.url=jdbc:h2:~/phoneshop/db/h2
```

Comment:
Are you using the same db file for test and production database? Please use different or use in memory db for tests